### DISCUSSION TITLE
Hi installed on Grafana 8.5.13 and getting this error

### DISCUSSION THREAD

**Discussion by chkp-oferb** [mullerpeter/databricks-grafana#32] [General]
Error: Minified React error #130; visit https://reactjs.org/docs/error-decoder.html?invariant=130&args[]=undefined&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.

**Comment by mullerpeter** [comment]
Hi @oferbd9

I was able to reproduce the error, it appears only in Grafana Versions < 9 (in 9.x & 10.x the error does not appear). The error is linked to the SecretField component used in the Datasource configuration UI for the Databricks Token.
Since this is a default Component provided by Grafana, I opened an issue in the grafana repo https://github.com/grafana/grafana/issues/75758

**Comment by mullerpeter** [comment]
In the meantime earlier releases should work for your version. Try using the v1.1.7 release of the plugin https://github.com/mullerpeter/databricks-grafana/releases/tag/v1.1.7
